### ISSUE TITLE
Image.Repository is not set by default anymore

### ISSUE THREAD

**Issue by BassFaceIV** [datawire/ambassador#3402]
**Describe the bug**
The chart depends on a value being set for image.repository. The documentation indicates it is set by default but 8 days ago the default was removed from values.yaml.

**To Reproduce**
(Updating a previous release)
1. kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml
2. helm upgrade ambassador datawire/ambassador

**Expected behavior**
All services should be properly updated and begin operation with K8's status "Running".

**Versions (please complete the following information):**
 - Ambassador: 1.13.4
 - Ambassador Chart: 6.7.6
 - Kubernetes environment: Microsoft AKS
 - Version: 1.18.14

**Additional context**
kubectl get pods -n ambassador

NAME                        |                      READY |  STATUS        |     RESTARTS  | AGE
-----------------------|------------------------|---------------|---------------|------
edge-stack-ambassador-85f6b9894b-r969w    |        1/1  |   Running    |        0      |    19m
edge-stack-ambassador-85f6b9894b-tpgll      |      1/1   |  Running     |       0     |     21m
edge-stack-ambassador-85f6b9894b-vw6k2    |        1/1   |  Running     |       0      |    20m
edge-stack-ambassador-agent-59c7c6fd5f-fwmmm   |   1/1 |    Running   |         0     |     21m
edge-stack-ambassador-injector-696cdc986-2dt8m |   0/1  |   InvalidImageName |  0   |       21m
edge-stack-ambassador-injector-6dfddc67dc-fmfmc |  1/1 |    Running    |        0     |     19d
edge-stack-ambassador-redis-56d7f997c6-lj8zv    |  1/1  |   Running     |       0     |     71d
telepresence-proxy-5cff68f75c-v4mw4        |       1/1   |  Running      |      0       |   49d
telepresence-proxy-d78b9784f-gbkkd         |       0/1   |  InvalidImageName  | 0   |       21m
traffic-manager-765f4bb59f-5rgfr           |       1/1   |  Running      |      0    |      49d

kubectl describe pods -n ambassador telepresence-proxy-d78b9784f-gbkkd

Events:
  Type   |  Reason     |    Age     |          From          |     Message
  -------|-------------|----------|--------------------|---------------
  Normal  | Scheduled   |   47s       |        default-scheduler |  Successfully assigned ambassador/telepresence-proxy-d78b9784f-gbkkd to aks-system-26662148-vmss000000
  Warning | InspectFailed |  1s (x6 over 42s) |  kubelet     |       Failed to apply default image tag ":1.13.4": couldn't parse image reference ":1.13.4": invalid reference format
  Warning |  Failed    |     1s (x6 over 42s) |  kubelet      |      Error: InvalidImageName

**Comment by acookin** [comment]
Hi @BassFaceIV 
I wasn't able to reproduce by upgrading from the previous chart version:

```
$ helm install ambassador -n ambassador --version 6.7.5 datawire/ambassador
$ helm repo update
$ helm upgrade ambassador -n ambassador datawire/ambassador
$ helm list -n ambassador
NAME      	NAMESPACE 	REVISION	UPDATED                             	STATUS  	CHART           	APP VERSION
ambassador	ambassador	2       	2021-05-12 09:19:05.189407 -0400 EDT	deployed	ambassador-6.7.6	1.13.4
$ kubectl get po -n ambassador -l app.kubernetes.io/name=ambassador
NAME                        READY   STATUS    RESTARTS   AGE
ambassador-844f5dc4-zp7jr   1/1     Running   0          5m16s
ambassador-844f5dc4-n9l6t   1/1     Running   0          4m15s
ambassador-844f5dc4-psq2g   1/1     Running   0          3m43s
$ kubectl get po -n ambassador ambassador-844f5dc4-zp7jr -o jsonpath='{.spec.containers[0].image}'
docker.io/datawire/aes:1.13.4
```

Can you let me know what your datawire helm repo's URL is (from the output of `helm repo list`)?

**Comment by BassFaceIV** [comment]
Thanks for taking a look @acookin.

The helm repo we're using is https://www.getambassador.io.

**Comment by acookin** [comment]
OH, sorry i misread that graph the first time. let me go fix that up. thanks!

**Comment by acookin** [comment]
This should be fixed in chart version 6.7.7.
Thanks @BassFaceIV !
